grails-app/views/template/_audioTranscribeParams.gsp

```diff
@@ -1,8 +1,8 @@
 <div class="form-group">
     <div class="col-md-offset-3 col-md-6">
         <div class="checkbox">
+            <g:checkBox name="hideDefaultButtons" data-default="true"/>
```

Comment:
If you move the checkbox outside the label tag you need to add an attribute (for) to the label to link it to the control.  

Also IIRC the bootstrap styling assumes the checkbox input tag goes inside the label tag, presumably you have a reason for moving it though?